volmdlr/edges.py

```diff
@@ -278,6 +278,203 @@ def split(self, split_point):
                     self.__class__(split_point, self.end)]
 
 
+class BSplineCurve():
```

Comment:
I think `BSplineCurve` should inherit from `Edge`, `BSplineCurve2D` should just inherit from `BSplineCurve`, and `BSplineCurve3D` should just inherit from `Primitive3D` and `BSplineCurve`

Reply:
You have to add `Edge.__init__(self, start, end, name=name)` to `BSplineCurve`'s `__init__`.

Reply:
By the way, I missed it, but you should add `BSplineCurve.__init__(self, ...)` to `BSplineCurve2D` and `BSplineCurve3D`'s `__init__`.